### PR TITLE
[DOCS] PhysicsDirectSpaceState enhancements

### PR DESCRIPTION
**Pull request by ghost**
This brings the PhysicsDirectSpaceState docs up to the standard made with Physics2DDirectSpaceState in #15026. Also makes small modifications to Physics2DDirectSpaceState. cc @cbscribe

### CODE REVIEW

File: doc/classes/PhysicsDirectSpaceState.xml [hunk 1]
```diff
@@ -63,13 +63,13 @@
 			</argument>
 			<description>
 				Intersect a ray in a given space. The returned object is a dictionary with the following fields:
```
Comment (cbscribe) [inline doc/classes/PhysicsDirectSpaceState.xml]: Please change this to "Intersects"

File: doc/classes/PhysicsDirectSpaceState.xml [hunk 2]
```diff
@@ -82,10 +82,10 @@
 			</argument>
 			<description>
 				Check the intersections of a shape, given through a [PhysicsShapeQueryParameters] object, against the space. The intersected shapes are returned in an array containing dictionaries with the following fields:
```
Comment (cbscribe) [inline doc/classes/PhysicsDirectSpaceState.xml]: Checks

**Comment by homer666** [comment]
@cbscribe Done. I noticed that I forgot to make these changes to the Physics2DDirectSpaceState docs. Is it worth going back and making a new PR for that?

**Comment by cbscribe** [comment]
Yeah I didn't notice it on the other one because it wasn't part of your changes. Don't make a new PR, just add it to this one.

**Comment by homer666** [comment]
I ended up squashing the changes into the single commit. Would it have been correct to create a separate commit for the Physics2DDirectSpaceState changes? I can fix it all up without a problem.

**Comment by cbscribe** [comment]
That's totally fine. You don't have to stick to one commit -> one file.

**Pull request merged**